### ISSUE TITLE
Runnable Instance 520 Error

### ISSUE THREAD

**Issue by SamHasler** [cburgmer/csscritic#72]
The [runnable instance](http://code.runnable.com/VXJo7YUrGNMz4gnD/csscritic-1-2-0-in-action) link in the readme shows a 520 error. Could it be replaced with a simpler demo using static files hosted on github?

**Comment by cburgmer** [comment]
Good point, I was thinking about something similar today. Technically we want to offer a quick glimpse into the main principle of the library, give the user a feeling what the tool is about. The static site so far demos the UI, but after accepting all test cases there's no way to play around and "fail" a test. We would want to offer the user a way of meddling with the HTML. 

One solution for handling this in-page could be by maybe storing the edited HTML into local storage for the test page to pick up?

Any other ideas? Do you feel you have time to come up with something small?

**Comment by SamHasler** [comment]
Local storage sounds like a good idea. I think having buttons to put it in
various states with a live editable view of the HTML/CSS would be good.

I may be able to look at this before the new year.
